src/engine/shared/config_variables.h

```diff
@@ -439,6 +439,7 @@ MACRO_CONFIG_INT(SvHighBandwidth, sv_high_bandwidth, 0, 0, 1, CFGFLAG_SERVER, "U
 MACRO_CONFIG_STR(SvRegister, sv_register, 16, "1", CFGFLAG_SERVER, "Register server with master server for public listing, can also accept a comma-separated list of protocols to register on, like 'ipv4,ipv6'")
 MACRO_CONFIG_STR(SvRegisterExtra, sv_register_extra, 256, "", CFGFLAG_SERVER, "Extra headers to send to the register endpoint, comma separated 'Header: Value' pairs")
 MACRO_CONFIG_STR(SvRegisterUrl, sv_register_url, 128, "https://master1.ddnet.org/ddnet/15/register", CFGFLAG_SERVER, "Masterserver URL to register to")
+MACRO_CONFIG_STR(SvMapsBaseUrl, sv_maps_base_url, 128, "", CFGFLAG_SERVER, "Client maps base URL (HTTPS)")
```

Comment:
```suggestion
MACRO_CONFIG_STR(SvMapsBaseUrl, sv_maps_base_url, 128, "", CFGFLAG_SERVER, "HTTPS base path for clients to download their maps. If the config is `https://a/` and the map name is `dm1`, then the URL is going to be `https://a/dm1_0b0c481d77519c32fbe85624ef16ec0fa9991aec7367ad538bd280f28d8c26cf.map`")
```

Somehow, this should document where the server admin should place their maps.

Reply:
@Robyt3 totally off topic but I really do miss more space here. Some configs could just use more text to properly document them.